testcafe/tests/photos_crud.js

```diff
@@ -0,0 +1,255 @@
+import { Selector, Role } from "testcafe"; //import testcafe function
+import { regularUser } from "./helpers/roles"; //import roles for login
+import config from "../config"; //import url & psswd config
+//import { generateRandomInteger } from "./helpers/utils.js";
+const _ = require("lodash");
```

Comment:
Can we use only one system to use external code? ie use import instead of require. And to make the thing right, we onyl want to import random not all the lib. 

Something like 
```
import random from 'lodash/random';
```